### PR TITLE
Update composer.json for latest sapi client

### PR DESCRIPTION
**Pull request by pivnicek**
Required for https://github.com/keboola/input-mapping/pull/68

Pending sapi-client release https://github.com/keboola/storage-api-php-client/releases/tag/untagged-3d86260daac18b393369

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     },
     "require": {
         "php": ">=5.6",
-        "keboola/storage-api-client": "10.21",
+        "keboola/storage-api-client": "10.22",
```
Comment (pivnicek) [inline composer.json]:
```suggestion
        "keboola/storage-api-client": "11.2",

```

File: composer.json [hunk 2]
```diff
@@ -27,7 +27,8 @@
     },
     "require": {
         "php": ">=5.6",
-        "keboola/storage-api-client": "10.21",
+        "keboola/storage-api-client": "11.2",
```
Comment (odinuv) [inline composer.json]:
```suggestion
        "keboola/storage-api-client": "^11.2",

```

**Review by odinuv — APPROVED** [review]

**Comment by pivnicek** [comment]
@odinuv what should we call this, `0.0.3`?

**Comment by pivnicek** [comment]
hmm, test failed on "branch not found".  is there a possible concurrency issue?

**Comment by odinuv** [comment]
they don't run in parallel, it's quite suspicous

**Comment by odinuv** [comment]
feel free to make it 1.0.0 :) it's already in production

**Pull request merged**